sdk/typescript/test/e2e/signature-mpc.test.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) dWallet Labs, Inc.
+// SPDX-License-Identifier: BSD-3-Clause-Clear
+
+import { beforeAll, describe, it } from 'vitest';
+
+import { launchProofMPSession } from '../../src/signature-mpc/proof';
+import { setup, TestToolbox } from './utils/setup';
+
+describe('Test signature mpc', () => {
+	let toolbox: TestToolbox;
+
+	beforeAll(async () => {
+		toolbox = await setup();
+	});
+
+	it('should create proof MPC Event', async () => {
+		await launchProofMPSession(toolbox.keypair, toolbox.client);
+		console.log(toolbox.keypair.toPeraAddress());
+	});
+});
```

Comment:
Suggestion for tests:
1. Try to replay a proof.
2. Try to replay a proof of another user.
3. Send an invalid proof. Check the user is still charged.

Reply:
This code will be replaced by DKG, PreSign & Sign, no need for all of these tests currently.